search/trim/trim.md

```diff
@@ -0,0 +1,125 @@
+# Trim modules
+
+The trim modules are six modules that modify given string-based enumerated values in various ways. For example, you can use `trimleft` to remove a given list of characters from the beginning of a given enumerated value.
+
+The trim modules consist of:
+
+- `trim`: Remove leading and trailing Unicode code points from an enumerated value.
+- `trimleft`: Remove leading Unicode code points from an enumerated value.
+- `trimprefix`: Remove a given prefix from an enumerated value.
+- `trimright`: Remove trailing Unicode code points from an enumerated value.
+- `trimspace`: Remove all leading and trailing whitespace from an enumerated value.
+- `trimsuffix`: Remove a given suffix from an enumerated value.
+
+## Supported Options
+
+None of the trim modules have any flags.
+
+## Overview
```

Comment:
I would recommend that we remove this Overview heading and use H2 formatting for each of the trim modules listed. This is also consistent with the Math Modules page. And adjust the syntax and example headings accordingly.

Reply:
Done